test/processors/concerns/http_files_test.rb

```diff
@@ -21,15 +21,25 @@ def logger
     WebMock.allow_net_connect!
   }
 
-  let(:uri) { URI.parse('http://test.prx.org/test/file.mp2') }
+  let(:http_uri) { URI.parse('http://test.prx.org/test/file.mp2') }
+  let(:https_uri) { URI.parse('https://test.prx.org/test/file.mp2') }
 
   it 'can download from http url' do
 
-    stub_request(:get, uri.to_s).
+    stub_request(:get, http_uri.to_s).
       with(headers: {'Host'=>'test.prx.org:80'}).
       to_return(status: 200, headers: {}, body: File.open(in_file('test_short.mp2')))
 
-    tmp = processor.http_download_file(uri)
+    tmp = processor.http_download_file(http_uri)
+    tmp.must_be_instance_of File
+  end
+
+  it 'can download from https url' do
+    stub_request(:get, https_uri.to_s).
+      with(:headers => {'Host'=>'test.prx.org:443'}).
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.